module/VuFind/src/VuFindTest/Feature/AutoRetryTrait.php

```diff
@@ -107,13 +107,18 @@ public function runBare(): void
                 // Execute callbacks for interrupted test, unless this is the
                 // last round of testing:
                 if ($this->retriesLeft > 0) {
-                    $this->logWarning(
-                        'RETRY ' . $this->getName(false)
-                        . " ({$this->retriesLeft} left)."
-                        . ' See PHP error log for details.',
-                        'RETRY TEST ' . $this->getName(false) . ' after exception: '
-                        . $e->getTraceAsString()
-                    );
+                    $logMethod = $annotations['method']['retryLogMethod']
+                        ?? [$this, 'logWarning'];
```

Comment:
Should this actually be:

<pre>
$logMethod = [$this, $annotations['method']['retryLogMethod'] ?? 'logWarning'];
</pre>

? Otherwise, the annotation is more `logFunction` than `logMethod`, unless I'm missing something.

Reply:
Should have tested.. Also, annotations are arrays, so needed to take the first element. Fixed and tested.